src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionIntegrationTest.java

```diff
@@ -1430,6 +1433,131 @@ public void failWrongParameterType() {
           .body("errors[0].errorCode", is("INVALID_CREATE_COLLECTION_OPTIONS"))
           .body("errors[0].exceptionClass", is("JsonApiException"));
     }
+
+    @Test
+    public void failNotProvideRequiredModelParameters() {
+      given()
+          .headers(getHeaders())
+          .contentType(ContentType.JSON)
+          .body(
+              """
+                            {
+                                "createCollection": {
+                                    "name": "collection_with_vector_service",
+                                    "options": {
+                                        "vector": {
+                                            "metric": "cosine",
+                                            "dimension": 768,
+                                            "service": {
+                                                "provider": "vertexai",
+                                                "modelName": "textembedding-gecko@003",
+                                                "parameters": {
+                                                    "projectId": "test"
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                            """)
+          .when()
+          .post(NamespaceResource.BASE_PATH, namespaceName)
+          .then()
+          .statusCode(200)
+          .body("status", is(nullValue()))
+          .body("data", is(nullValue()))
+          .body(
+              "errors[0].message",
+              startsWith(
+                  "The provided options are invalid: Required parameter 'autoTruncate' for the provider 'vertexai' missing"))
+          .body("errors[0].errorCode", is("INVALID_CREATE_COLLECTION_OPTIONS"))
```

Comment:
I copied the structure from previous tests, but fix it

Reply:
Yeah not a big deal -- I started doing it myself when I noticed that it makes sense to start with errorCode that rarely changes and only then message that changes more often. So that way kind of get more tested.

But it is also personal preference, not a hard guideline.